Source/Scene/Model.js

```diff
@@ -654,6 +654,8 @@ define([
         this._rtcCenterEye = undefined; // in eye coordinates
         this._rtcCenter3D = undefined;  // in world coordinates
         this._rtcCenter2D = undefined;  // in projected world coordinates
+
+        this._keepPipelineExtras = options.keepPipelineExtras; // keep the buffers in memory for use in other applications
```

Comment:
Default value of `false`?

Reply:
There's no need.  `false` and `undefined` both evaluate to `false` in the if statement.  I'm not sure if we have a coding standard for this for this repo, but on some of the other projects I work on we usually don't bother default valuing to false.